docs/installation.md

```diff
@@ -1,30 +1,65 @@
-# Stable
+# Install From Package
 
-To get started with Websurfx, clone the repository, edit the config file which is located in the `websurfx` directory and install redis server by following the instructions located [here](https://redis.io/docs/getting-started/) and then run the websurfx server and redis server using the following commands:
+## Arch Linux
+
+You can install `Websurfx` through the [Aur](https://aur.archlinux.org/packages/websurfx-git), Currently we only support `Rolling/Edge` version. You can install the rolling/edge version by running the following command (using [paru](https://github.com/Morganamilo/paru)):
+
+```bash
+paru -S websurfx-edge-git
+```
+
+After installing it you can run the websurfx server by running the following commands:
+
+``` bash
+redis-server --port 8082 &
+websurfx
+```
+
+Once you have started the server, open your preferred web browser and navigate to http://127.0.0.1:8080/ to start using Websurfx.
+
+If you want to change the port or the ip or any other configuration setting checkout the [configuration docs](./configuration.md).
+
+## Other Distros 
+
+The package is currently not available on other Linux distros. With contribution and support it can be made available on other distros as well 🙂.
+
+# Install From Source
+
+Before you can start building `websurfx`, you will need to have `Cargo` installed on your system. You can find the installation instructions [here](https://doc.rust-lang.org/cargo/getting-started/installation.html).
+
+## Stable
+
+To get started with Websurfx, clone the repository, edit the config file which is located in the `websurfx` directory and install redis server by following the instructions located [here](https://redis.io/docs/getting-started/) and then build and run the websurfx server by running the following commands:
 
 ```shell
 git clone https://github.com/neon-mmd/websurfx.git
 cd websurfx
-cargo build
+cargo build -r
 redis-server --port 8082 &
 ./target/debug/websurfx
 ```
 
-# Rolling/Edge/Unstable
+Once you have started the server, open your preferred web browser and navigate to http://127.0.0.1:8080/ to start using Websurfx.
+
+If you want to change the port or the ip or any other configuration setting checkout the [configuration docs](./configuration.md).
+
+## Rolling/Edge/Unstable
 
 If you want to use the rolling/edge branch, run the following commands instead:
 
 ```shell
 git clone https://github.com/neon-mmd/websurfx.git
 cd websurfx
 git checkout rolling
-cargo build
+cargo build -r
 redis-server --port 8082 &
 ./target/debug/websurfx
```

Comment:
Same as the above

Reply:
Also this one as well I just overlooked it, I think next time I will have to be more careful :slightly_smiling_face: .